src/components/dev-hub/article-share-footer.js

```diff
@@ -30,7 +30,7 @@ const BlogShareLinks = styled('div')`
     }
 `;
 
-const ArticleShareFooter = ({ tags, title, url, ...props }) => {
+const ArticleShareFooter = ({ tags, title, url, tooltipText, ...props }) => {
```

Comment:
![Screen Shot 2021-05-17 at 1 21 15 PM](https://user-images.githubusercontent.com/9064401/118530587-cfe33380-b712-11eb-8ee2-485cefb4e70c.png)

Perhaps we should pull in the expose article share links PR here as well, since this is going off of the screen on mobile. I believe that PR should fix this.

Reply:
Yeah, let's pull a new tooltip for this PR as well.

Reply:
Okay great, I am going to clean up #823 based on your comments and then we can use that perhaps here. For now we can continue and then add that in later.